javasource/audittrail/log/CreateLogObject.java

```diff
@@ -189,6 +192,13 @@ public static IMendixObject createAuditLogItems(final IMendixObject auditableObj
 		}
 	}
 
+	private static String getTimeZone(final IContext context) {
+		final TimeZone tz = context.getSession().getTimeZone();
+		if (tz != null) { return tz.getID(); }
+		if (Constants.getServerTimeZone() != null && !Constants.getServerTimeZone().isEmpty()) { return Constants.getServerTimeZone(); }

```

Comment:
did not know it existed in Java :)

Reply:
Fair enough. Although I would consider swapping the values, so that the exceptional cases are early exits and the normal case is at the end:

```java
if (Constants.getServerTimeZone() == null || Constants.getServerTimeZone().isEmpty()) return "GMT";
return Constants.getServerTimeZone();
```